firmware/ventilator-controller-stm32/Core/Inc/Pufferfish/Driver/I2C/HoneywellABP/Device.h

```diff
@@ -32,23 +31,20 @@ class Device : public Testable {
    */
   I2CDeviceStatus read_sample(ABPSample &sample);
 
-  I2CDeviceStatus test() override;
-  I2CDeviceStatus reset() override;
-
  private:
   Pufferfish::HAL::Interfaces::I2CDevice &dev_;
 
+  static const uint8_t status_shift = 6;
+  static const size_t bridge_high = 0;
+  static const size_t bridge_low = 0;
+
   // pressure range (refer to datasheet)
   const float pmin;  // minimum pressure
   const float pmax;  // maximum pressure
```

Comment:
Do we need still these?

Reply:
Yes they are initialized in the constructor with the config from main.cpp, and used in the raw_to_pressure formula in `Device.cpp`

Reply:
Oh wait do you mean since we are using only one config? right